src/systems/filecoin_nodes/clock/_index.md

```diff
@@ -39,14 +42,13 @@ Computer-grade clock crystals can be expected to have drift rates on the order o
   - See how [go-ethereum does this](https://github.com/ethereum/go-ethereum/blob/master/p2p/discv5/ntp.go) for inspiration
 - clients CAN consider using cesium clocks instead for accurate synchrony within larger mining operations
 
-Assuming a majority of rational participants, the above should lead to relatively low skew over time, with seldom more than 10-20% clock skew that should be rectified periodically by the network, as is the case in other networks. This assumption can be tested over time by ensuring that:
-
-- (real-time) epoch time is as dictated by the protocol
-- (historical) the current epoch number is as expected
+Assuming a majority of rational participants, the above should lead to relatively low skew over time, with seldom more than 10-20% clock skew that should be rectified periodically by the network, as is the case in other networks. This assumption will be maintained over time with both the real-time -- blocks with timestamps outside of current epoch time are rejected -- and historical -- blocks with timestamps that don't match protocol specified epochs are rejected -- validation rules.
```

Comment:
assumption will be maintained makes it seem like less of an assumption. Do we know how to rectify? If not, the "tested" signifies we should be ensuring the assumption holds, else speccing a solution